### DISCUSSION TITLE
what it prefix of fdda?

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#8501] [Q&A] [answered]
Dear all,

I would like to consult what the prefix of 0xfdda is? I didn't found this prefix in thread spec, thank you.
```
MeshForwarder-: Sent IPv6 UDP msg, len:198, chksum:8a83, ecn:no, to:0x2000, sec:yes, prio:normal
MeshForwarder-:     src:[fdda:38c0:c5e:1:39cf:87c1:e035:605f]:5540
MeshForwarder-:     dst:[fdda:38c0:c5e:1:8e5a:4609:962c:f312]:5540
```

**Comment by jwhui** [comment, marked as answer]
It is a randomly generated IPv6 Unique Local Prefix for communicating with IPv6 endpoints that are outside the Thread network. See the [Thread Border Router codelab](https://openthread.io/codelabs/openthread-border-router#2) for more information.

**Reply by Leo2442926161** [reply]
Thank you Jwhui,

What's the difference between LLA and ML-EID?

These are local IPv6 address as well, where can I get explaination about this?
```
sudo ot-ctl ipaddr  
fda8:5ce9:df1e:6620:0:ff:fe00:fc11
fda8:5ce9:df1e:6620:0:0:0:fc38
fda8:5ce9:df1e:6620:0:ff:fe00:fc10
fd76:a5d1:fcb0:1707:f3c7:d88c:efd1:24a9
fda8:5ce9:df1e:6620:0:ff:fe00:fc00
fda8:5ce9:df1e:6620:0:ff:fe00:4000
fda8:5ce9:df1e:6620:3593:acfc:10db:1a8d
```

**Reply by jwhui** [reply]
> What's the difference between LLA and ML-EID?

From the [Thread Primer](https://openthread.io/guides/thread-primer/ipv6-addressing):
- Link-Local — all interfaces reachable by a single radio transmission
- Mesh-Local — all interfaces reachable within the same Thread network

The `fda8::` prefix is a Mesh-Local Prefix. There are multiple Mesh-Local IPv6 addresses (unicast vs anycast, EID vs. RLOC, etc.).

**Reply by Leo2442926161** [reply]
Dear Jwhui, Thank you again,

`It is a randomly generated IPv6 Unique Local Prefix `
What's the difference between this Local address and LLA, ML-EID?

the Mesh-Local says Always has a prefix fd00::/8, but those are faa8 and fdda, is there other clarify?
![image](https://user-images.githubusercontent.com/30258768/206330373-cda1bcc7-29fa-46d2-81a2-f7356ab0d387.png)

**Reply by jwhui** [reply]
> What's the difference between this Local address and LLA, ML-EID?

I described the difference in https://github.com/openthread/openthread/discussions/8501#discussioncomment-4338535.

> The Mesh-Local says Always has a prefix fd00::/8, but those are faa8 and fdda, is there other clarify?

All IPv6 Unique Local Prefixes in Thread have the same `fd00::/8` prefix. Both Mesh-Local and Off-Mesh Routable prefixes are IPv6 Unique Local Prefixes.

**Reply by Leo2442926161** [reply]
Thank you Jwhui
